app/controllers/solidus_paypal_braintree/client_tokens_controller.rb

```diff
@@ -18,5 +18,12 @@ def load_gateway
         @gateway = ::SolidusPaypalBraintree::Gateway.where(active: true).merge(store_payment_methods_scope).first!
       end
     end
+
+    def generate_token
+      @gateway.generate_token
+    rescue ::SolidusPaypalBraintree::Gateway::TokenGenerationDisabledError => error
+      Rails.logger.warn error
```

Comment:
Shouldn't be `Rails.logger.error`?

Reply:
I wasn't sure what level it should be. I'm ok changing it to `Rails.logger.error`